src/App/App.js

```diff
@@ -15,11 +15,8 @@ const App = (props) => {
         <Switch>
           <Route exact path='/' component={Home}/>
           <Route exact path='/packing-quiz' component={Quiz}/>
-          <Route exact path='/saved-packing-lists' component={SavedView}/>
-          
-          {/* Temporary Route while building PackingList Component */}
           <Route exact path='/packing-list' component={PackingList}/>
```

Comment:
We'll need to do some sort of error handling for this in the event that a user navigates to PackingList from the URL rather the quiz.